src/buildHTML.js

```diff
@@ -1491,6 +1491,22 @@ groupTypes.mclass = function(group, options) {
     return makeSpan([group.value.mclass], elements, options);
 };
 
+groupTypes.transform = function(group, options) {
+    const span = groupTypes.text(group, options);
+    if (group.value.dy) {
+        const dy = group.value.dy.value;
+        span.style.top = "" + (-dy.number) + dy.unit;
```

Comment:
Good point -- indeed, it currently does not, but would be easy to tweak to.  (A bigger issue is that `calculateSize` only supports mu and em.)